internal/ethapi/api.go

```diff
@@ -1176,6 +1176,10 @@ func DoEstimateGas(ctx context.Context, b Backend, args TransactionArgs, blockNr
 		if err != nil {
 			return 0, err
 		}
+		err = overrides.Apply(state)
```

Comment:
overrides maybe nil, we should check before use.

Reply:
it gets checked in the `api.go:919`